docs/features.md

```diff
@@ -41,8 +41,61 @@ Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Ob
 the cluster into Monokle - [read more](cluster-integration.md). If you need to make any immediate hot-fixes simply modify
 the resource in the source editor and click the "Apply" button.
 
+## Cluster Compare
 
+Cluster Compare functionality in Monokle 1.3.0 allows you to compare a set of local resources (plain or generated with Kustomize/helm) against a cluster. 
 
+![Cluster Compare](img/clustercompare-1.png)
 
+## The Improved Validation Logic and Error Display
 
+In the latest Monokle 1.3.0, all the errors can be found at the top of the resource navigator to view the invalid resources.  
 
+![The Improved Validation Logic and Error Display](img/resource-navigator.png)
+
+Selecting a resource from the list will automatically open it for editing in the editor.  
+[The Improved Validation Logic and Error Display](img/resourcenavigator.gif)
+
+## Handling Multiple KubeConfig Contexts
+
+Supposedly, the chosen Kubernetes file contains numerous contexts. Monokle 1.3.0 will find those and allow you to select the right one for your cluster operations. 
```

Comment:
"If the selected kubeconfig file contains numerous contexts, Monokle will find those..."